tests/test_ax_integration.py

```diff
@@ -57,12 +70,14 @@ def simulate_response(trial_params):
 
         cls.df = exp_to_df(cls.client.server.strat.experiment)
 
-        cls.config = Config(config_fnames=[config_file])
+        cls.config = Config(config_fnames=[cls.config_file])
 
+    @ignorable
     def tearDown(self):
         if self.client.server.db is not None:
             self.client.server.db.delete_db()
 
+    @ignorable
```

Comment:
The fact that [a] you have to decorate every method in the class with this thing and [b] it's hard to understand what it does suggests to me that this is not the best approach here. Explicit is better than implicit, and clear is better than clever. If you can explain what you're trying to do here I wonder if we can come up with a better solution.

Reply:
Indeed, it turns out I should have just raised the skip test exception to fulfil what I wanted. I just issued a fix.